### ISSUE TITLE
Duplicating guide books

### ISSUE THREAD

**Issue by WitherKing43** [TangoTek/TekTopia-Community#916]
**Describe the bug**
Every time I open my personal save of tektopia (in-game world) i get a new TekTopia Guide Book. I think it just cloggs up inventory

**To Reproduce**
simply start a new world with tektopia loaded. then you simply save and quit then come back into the world and you should have a new guide book. 
**Expected behavior**
if it is possible. make the mod scan chests/inventories to see if there is already is a guide book

**Screenshots**

** Server/Location **
Im on my own survival world with the mod loaded

**Additional context**

**Comment by TangoTek** [comment]
Just tested this in a clean new world with just tektopia loaded. It didn't give me a new book each time I logged in. 
Can you disable any other mods you have running and try again?  
You aren't restoring a backup each time or something are you?

**Comment by TheJamesRyan** [comment]
I'm getting the duplication glitch as well. It only seems to happen when I save and quite without a guide book in my inventory.

**Comment by The-Intelligent-One** [comment]
Yeah, same here, I keep having to trash the book. It's a minor annoyance, but should still probably be fixed.

**Comment by TheJamesRyan** [comment]
It just happened again, hasn't happened since I posted last. This time though, the duplicate book was not in my inventory, but waiting for me with my original book where I died. It was just laying there next to it on the ground.

**Comment by ptrxyz** [comment]
This also happens to me on 1.12.2/Tektopia Version 1.1.0.

Anyway to disable the guide completly maybe?
